src/Toolkit.Forms/OverviewMap/OverviewMap.cs

```diff
@@ -0,0 +1,226 @@
+﻿// /*******************************************************************************
+//  * Copyright 2012-2018 Esri
+//  *
+//  *  Licensed under the Apache License, Version 2.0 (the "License");
+//  *  you may not use this file except in compliance with the License.
+//  *  You may obtain a copy of the License at
+//  *
+//  *  http://www.apache.org/licenses/LICENSE-2.0
+//  *
+//  *   Unless required by applicable law or agreed to in writing, software
+//  *   distributed under the License is distributed on an "AS IS" BASIS,
+//  *   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  *   See the License for the specific language governing permissions and
+//  *   limitations under the License.
+//  ******************************************************************************/
+
+using Esri.ArcGISRuntime.Mapping;
+using Esri.ArcGISRuntime.Symbology;
+using Esri.ArcGISRuntime.Toolkit.UI.Controls;
+using Esri.ArcGISRuntime.Xamarin.Forms;
+using Xamarin.Forms;
+using Xamarin.Forms.Xaml;
+
+namespace Esri.ArcGISRuntime.Toolkit.Xamarin.Forms
+{
+    /// <summary>
+    /// Defines a small "overview" (or "inset") map displaying a representation of the attached <see cref="GeoView"/>'s current viewpoint.
+    /// </summary>
+    public class OverviewMap : TemplatedView
+    {
+        private OverviewMapController? _controller;
+        private static readonly ControlTemplate DefaultControlTemplate;
+
+        private MapView? _overviewMapView;
+
+        static OverviewMap()
+        {
+            string template = @"<ControlTemplate xmlns=""http://xamarin.com/schemas/2014/forms""
+                                                 xmlns:x=""http://schemas.microsoft.com/winfx/2009/xaml""
+                                                 xmlns:esri=""clr-namespace:Esri.ArcGISRuntime.Xamarin.Forms;assembly=Esri.ArcGISRuntime.Xamarin.Forms""
+                                                 xmlns:internal=""clr-namespace:Esri.ArcGISRuntime.Toolkit.Xamarin.Forms.Internal;assembly=Esri.ArcGISRuntime.Toolkit.Xamarin.Forms"">
+                                    <Frame Padding=""1"" WidthRequest=""100"" HeightRequest=""100"" CornerRadius=""0"" HasShadow=""False"" BorderColor=""Black"" BackgroundColor=""White"">
```

Comment:
Should this size be hardcoded here? Wouldn't that mean a larger size around it won't make it stretch?